src/targets/gpu/include/migraphx/gpu/gemm.hpp

```diff
@@ -132,6 +133,43 @@ struct rocblas_gemm
     {
         return shapes.size() - 1;
     }
+
+    void finalize(context& ctx, const shape& output_shape, const std::vector<shape>& input_shapes)
+    {
+        rocblas_initialize();
```

Comment:
We should call `rocblas_initialize` in get_rocblas in the context.

Reply:
done.